be/src/exec/vectorized/hash_join_node.cpp

```diff
@@ -98,6 +98,11 @@ Status HashJoinNode::init(const TPlanNode& tnode, RuntimeState* state) {
     if (tnode.__isset.need_create_tuple_columns) {
         _need_create_tuple_columns = tnode.need_create_tuple_columns;
     }
+
+    if (tnode.hash_join_node.__isset.output_columns) {
+        _output_slots.insert(_output_slots.end(), tnode.hash_join_node.output_columns.begin(),
+                             tnode.hash_join_node.output_columns.end());
+    }
```

Comment:
can this stmt `_output_slots = tnode.hash_join_node.output_columns;` replace tedious insert stmt?

Reply:
_output_slots has modify type from vector to set